partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.6/WhitelistedTests.txt

```diff
@@ -3070,3 +3070,9 @@ run/virtpatmat_switch.scala
 run/t5629b.scala
 run/t6318_primitives.scala
 run/t8764.scala
+
+# Big numerics added
```

Comment:
The last section of this file has some tests blocked by comments so if I just append these they will look like part of a different set. Should I remove the other comments/blank lines for consistency?